### PR TITLE
Adds ability to have multiple shabadIDs in the query

### LINKED ISSUE #28: support IDs query string
ie IDs=2188,3700,4427,6279,6357,14952,15422,15426,16162,18788,20012,21465,21468,22017,31114,38754,39324,40271,41556,42657,44002,44005,44148,44300,44339,45663,47891,47897,56392,56702,59785

reference: https://github.com/KhalisFoundation/SikhiToTheMax-Web/issues/67#issuecomment-374001099

### PR DESCRIPTION
**Pull request by ManjotS**
fixes #28

### CODE REVIEW

File: api/lib/index.js [hunk 1]
```diff
@@ -18,6 +18,20 @@ module.exports = {
       },
     });
   },
+  isListOfNumbers: str => {
+    if (typeof str !== 'string' || Number.isNaN(str)) {
```
Comment (tsingh777) [inline api/lib/index.js]: the second condition will always be false if the first one is false. do you need the isNaN check anymore? the regex will catch non numbers.

Comment (ManjotS) [reply api/lib/index.js]: want it to catch things that are neither strings nor numbers.

Comment (tsingh777) [reply api/lib/index.js]: you don't need the `isNaN`.` typeof str !== 'string'` catches everything that is not a string. 
image

**Comment by ManjotS** [comment]
So that means numbers would return false . We don't want that..

**Pull request merged**